tests/components/picnic/conftest.py

```diff
@@ -50,3 +54,42 @@ async def init_integration(
     await hass.async_block_till_done()
 
     return mock_config_entry
+
+
+@pytest.fixture
+def ws_req_id() -> Callable[[], int]:
```

Comment:
We have a common helper on the websocket client for sending messages with incrementing ids, `client.send_json_auto_id`.

https://github.com/home-assistant/core/blob/670e5a2eae0a49ffd0552b7171bac9f2b363b20f/tests/components/demo/test_weather.py#L142